nautobot/ipam/forms.py

```diff
@@ -235,7 +238,8 @@ class RIRFilterForm(BootstrapMixin, CustomFieldFilterForm):
 #
 
 
-class AggregateForm(BootstrapMixin, TenancyForm, CustomFieldModelForm, RelationshipModelForm):
+class AggregateForm(BootstrapMixin, TenancyForm, PrefixFieldMixin, CustomFieldModelForm, RelationshipModelForm):
+    prefix = IPNetworkFormField()
```

Comment:
Same feedback here as with `nautobot.dcim.Device.primary_ip4/6` and all other explicit uses of `IPNetworkFormField`. This could go away I believe once `nautobot.ipam.fields.VarbinaryIPField.formfield()` is defined.